### ISSUE TITLE
question about  test Accuracy

### ISSUE THREAD

**Issue by SaintLogos1234** [shun-zheng/Doc2EDAG#5]
I ran the code you provided to train the model of DCFEE，I test it, I find that the accuracy is quite different from that provided by your paper, the follow is my result:
![image](https://user-images.githubusercontent.com/19261529/69520310-46c4c380-0f97-11ea-8d63-c0ba6d67bf58.png)
my training seting is:
![image](https://user-images.githubusercontent.com/19261529/69520394-8db2b900-0f97-11ea-8b08-28a89138b0aa.png)
Why is this so？

**Comment by shun-zheng** [comment]
Because you only evaluate the performance of DCFEE models.

**Comment by SaintLogos1234** [comment]
I mean, why is the performance of my DCFEE model so different from that of your paper，For example, in your paper，DCFEE-O：EF:P:66.0,R:41.6,F1:51.1,but my test result is :60.4/32.0/41.8,why

**Comment by SaintLogos1234** [comment]
I have a question, because of my limited resources, I reduced your training data, test data, dev data. The data for each training and test set and dev set was reduced to: 800,100,100. my total traing data is :4000, test data and training data is 500, 500; Do I need to modify any hyperparameters？I conducted the experiment according to the hyperparameters you provided, and found that the GPU memory-usage is high, but the Utilization rate is low，under 30%,Do you have any advice？

**Comment by SaintLogos1234** [comment]
I have other question: In your code,In DCFEE document-level event extraction section, the input of event dection is sentence embedding, no concatenated with SEE output, that is different with DCFEE,In that paper, the input of event dection is setence embedding concatened with SEE output, Why is this different?

**Comment by shun-zheng** [comment]
Thanks for your interests in our work.

I hope the following answers may address your concerns.

1. Training with only a small proportion of data will definitely hurt the final performance. This may explains that you models cannot beat the baselines reported in the paper.

2. According to our experience, for the same batch size (such as 64), utilizing multiple GPUs can obtain better performance than using a single GPUs with the 'gradient accumulation steps' option. 
Currently, I cannot explain this observation very well, but I do observe it. 

3. In our study, we provide DCFEE with a much stronger encoder based on Transformer, which is the same to the one used in Doc2EDAG.
And we only implement the central idea of DCFEE, because we do not find open-source codes of the DCFEE paper.
Actually, we also do not know how to concatenate sentence embedding with those outputs derived from the same sequence to make a significant difference.
